views/partials/svg/glyphs/ip-addresses.hbs

```diff
@@ -1,4 +1,4 @@
 
 <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
-  <path fill="#ab71ff" fill-rule="evenodd" d="M24.0001658,5.3760452 C24.0002392,6.02041338 23.7357422,6.63715518 23.2673574,7.08477374 C22.7989726,7.5323923 22.1669056,7.7724646 21.5161693,7.74991171 L17.7193745,13.7611735 C18.1968363,14.6503574 18.0558172,15.7408496 17.3674748,16.4823927 C16.6791324,17.2239357 15.5930899,17.4553424 14.6573594,17.059846 C13.7216289,16.6643495 13.1396075,15.727918 13.2049807,14.7230704 L9.46458587,12.2553892 C9.10841452,12.4306037 8.71369143,12.5150954 8.31618746,12.5012073 L4.51939269,18.510094 C5.02926571,19.4592775 4.83134628,20.6290243 4.03684211,21.3620726 C3.24233794,22.0951209 2.04935806,22.2086857 1.12821853,21.6389567 C0.207079006,21.0692278 -0.212534717,19.9582666 0.105151518,18.9303042 C0.422837752,17.9023419 1.39860912,17.2137 2.4841955,17.2513154 L6.28099026,11.2400536 C5.8035285,10.3508697 5.9445476,9.26037748 6.63288997,8.51883443 C7.32123233,7.77729138 8.40727482,7.5458847 9.34300535,7.94138114 C10.2787359,8.33687758 10.8607572,9.27330914 10.795384,10.2781567 L14.5357789,12.7458379 C14.8920573,12.5710335 15.2867751,12.4869512 15.6841773,12.5012073 L19.4809721,6.48994554 C19.0121593,5.61716504 19.1385753,4.54828748 19.7983749,3.8062277 C20.4581745,3.06416792 21.5134111,2.80407098 22.4478433,3.15318117 C23.3822755,3.50229135 24.0005382,4.38761896 24.0001658,5.3760452 Z"/>
+  <path fill="#7542E5" fill-rule="evenodd" d="M24.0001658,5.3760452 C24.0002392,6.02041338 23.7357422,6.63715518 23.2673574,7.08477374 C22.7989726,7.5323923 22.1669056,7.7724646 21.5161693,7.74991171 L17.7193745,13.7611735 C18.1968363,14.6503574 18.0558172,15.7408496 17.3674748,16.4823927 C16.6791324,17.2239357 15.5930899,17.4553424 14.6573594,17.059846 C13.7216289,16.6643495 13.1396075,15.727918 13.2049807,14.7230704 L9.46458587,12.2553892 C9.10841452,12.4306037 8.71369143,12.5150954 8.31618746,12.5012073 L4.51939269,18.510094 C5.02926571,19.4592775 4.83134628,20.6290243 4.03684211,21.3620726 C3.24233794,22.0951209 2.04935806,22.2086857 1.12821853,21.6389567 C0.207079006,21.0692278 -0.212534717,19.9582666 0.105151518,18.9303042 C0.422837752,17.9023419 1.39860912,17.2137 2.4841955,17.2513154 L6.28099026,11.2400536 C5.8035285,10.3508697 5.9445476,9.26037748 6.63288997,8.51883443 C7.32123233,7.77729138 8.40727482,7.5458847 9.34300535,7.94138114 C10.2787359,8.33687758 10.8607572,9.27330914 10.795384,10.2781567 L14.5357789,12.7458379 C14.8920573,12.5710335 15.2867751,12.4869512 15.6841773,12.5012073 L19.4809721,6.48994554 C19.0121593,5.61716504 19.1385753,4.54828748 19.7983749,3.8062277 C20.4581745,3.06416792 21.5134111,2.80407098 22.4478433,3.15318117 C23.3822755,3.50229135 24.0005382,4.38761896 24.0001658,5.3760452 Z"/>
```

Comment:
Hard to see from diffs. I blindly (and foolishly) assume the only thing that we changed here was the `fill=""` color (here and all the other glyphs/*.hbs files).
I wonder if it's worth seeing if we can modify the SVG fill colors via CSS to avoid big diffs in the future (if we ever need to change the fill colors), or if that gets infinitely tricky because these are inlined SVGs via HBS.